docs: add nuxt-link defaults config in nuxt.config (#25610)
nuxt/nuxt: docs/3.api/1.components/4.nuxt-link.md

## Overwriting Defaults

> ### By Nuxt Config

Reviewer: ```suggestion
### In Nuxt Config
```

> You can overwrite `<NuxtLink>` defaults by [`nuxt.config`](https://nuxt.com/docs/api/nuxt-config#defaults)

Reviewer: ```suggestion
You can overwrite some `<NuxtLink>` defaults in your [`nuxt.config`](https://nuxt.com/docs/api/nuxt-config#defaults)
```

::callout{color="amber" icon="i-ph-warning-duotone"}
These options will likely be moved elsewhere in the future, such as into `app.config` or into the `app/` directory.
::

```ts [nuxt.config.ts]
export default defineNuxtConfig({
  experimental: {
    defaults: {
      nuxtLink: {

>         componentName?: string;
>         externalRelAttribute?: string;
>         activeClass?: string;
>         exactActiveClass?: string;
>         prefetchedClass?: string;
>         trailingSlash?: 'append' | 'remove'

Reviewer: ```suggestion
        // default values
        componentName: 'NuxtLink',
        externalRelAttribute: 'noopener noreferrer',
        activeClass: 'router-link-active',
        exactActiveClass: 'router-link-exact-active',
        prefetchedClass: undefined, // can be any valid string class name
        trailingSlash: undefined // can be 'append' or 'remove'
```

      }
    }
  }
})
```

> ### Recommended

Reviewer: ```suggestion
### Custom Link Component
```
